manifests/0700_cr.yaml

```diff
@@ -1,8 +1,26 @@
-apiVersion: nfd.kubernetes.io/v1alpha1
+apiVersion: nfd.kubernetes.io/v1
 kind: NodeFeatureDiscovery
 metadata:
   name: nfd-master-server
   namespace: REPLACE_NAMESPACE
 spec:
-  namespace: node-feature-discovery-operator
-  image: k8s.gcr.io/nfd/node-feature-discovery:v0.6.0
+  operand:
+    namespace: node-feature-discovery-operator
+    image: node-feature-discovery
+    repository: k8s.gcr.io/nfd
+    version: v0.6.0
```

Comment:
is there a use-case to keep these separate? My preference would be to keep just `image` . FWIW, I'm also not sure about `namespace` if the deployment gets created in the CR namespace.

Reply:
This granularity help us during testing, is almost the same I know, but I prefer to visualize it this way. I am ok with a single var holding the image info.

The namespace is there as a placeholder, I am not modifying this, current CR has image and namespace

Reply:
As I wrote, my preference is just `image`. This would be consistent with what pod spec has so less confusion for the user too.

Reply:
c9260dc

Reply:
does c9260dc address this requested change?  @mythi